sqlalchemydiff/comparer.py

```diff
@@ -335,6 +349,53 @@ def _get_columns(inspector, table_name):
     return inspector.get_columns(table_name)
 
 
+def _get_constraints_info(left_inspector, right_inspector,
+                          table_name, ignores):
+    left_constraints_list = _get_constraints_data(left_inspector, table_name)
+    right_constraints_list = _get_constraints_data(right_inspector, table_name)
+
+    left_constraints_list = _discard_ignores_by_name(left_constraints_list,
+                                                     ignores)
+    right_constraints_list = _discard_ignores_by_name(right_constraints_list,
+                                                      ignores)
+
+    # process into dict
+    left_constraints = dict((elem['name'], elem)
+                            for elem in left_constraints_list)
+    right_constraints = dict((elem['name'], elem)
+                             for elem in right_constraints_list)
+
+    return _diff_dicts(left_constraints, right_constraints)
+
+
+def _get_constraints_data(inspector, table_name):
+    try:
+        return inspector.get_check_constraints(table_name)
+    except NotImplementedError:
+        return []
+
+
+def _get_enums_info(left_inspector, right_inspector, ignores):
+    left_enums_list = _get_enums_data(left_inspector)
+    right_enums_list = _get_enums_data(right_inspector)
+
+    left_enums_list = _discard_ignores_by_name(left_enums_list, ignores)
+    right_enums_list = _discard_ignores_by_name(right_enums_list, ignores)
+
+    # process into dict
+    left_enums = dict((elem['name'], elem) for elem in left_enums_list)
+    right_enums = dict((elem['name'], elem) for elem in right_enums_list)
+
+    return _diff_dicts(left_enums, right_enums)
+
+
+def _get_enums_data(inspector):
+    try:
+        return inspector.get_enums()
```

Comment:
perhaps we could add a comment here mentioning that this is specific to the postgres inspector?